R/get_pvol_se.R

```diff
@@ -0,0 +1,43 @@
+get_pvol_se_radar_mapping <- c(
+  "seang" = "angelholm", "seatv" = "atvidaberg", "sebaa" = "balsta",
+  "sehem" = "hemse", "sehuv" = "hudiksvall", "sekaa" = "karlskrona",
+  "sekrn" = "kiruna", "selek" = "leksand", "sella" = "lulea",
+  "seoer" = "ornskoldsvik", "seosd" = "ostersund", "sevax" = "vara"
+)
+
+get_pvol_se <- function(radar, time, ..., call = rlang::caller_env()) {
+  if (!radar %in% names(get_pvol_se_radar_mapping)) {
+    cli::cli_abort(
+      c(
+        x = "The radar {.val {radar}} is not found in the mapping for Swedish radars",
+        i = "Most likely this means an invalid odim code has been provided"
+      ),
+      call = call, class = "getRad_error_get_pvol_se_radar_not_found"
+    )
+  }
+  radar_name <- get_pvol_se_radar_mapping[radar]
+  url <- glue::glue('/area/{radar_name}/product/qcvol/{lubridate::year(time)}/{lubridate::month(time)}/{{lubridate::day(time)}}/radar_{radar_name}_qcvol_{strftime(time, "%Y%m%d%H%M", tz="UTC" )}.h5')
```

Comment:
I'd probably break this long string up with req_url_path_append(), file.path (Peter likes this one, I try to avoid it), or usage of glue `.sep`:

eg. 

```r

glue::glue("area","bewid","product","qcvol",2018,"04",16,"radar_bewid_qcvol_040417.h5", .sep = "/")

``

Reply:
That looks like a re-implementation of paste ;), I guess I do like the longer format strings as then it can be controlled through a package option :)

Reply:
Implemented options for control